### ISSUE TITLE
Push 1 in Bitwig 2 is not detected

### ISSUE THREAD

**Issue by JohannesTress** [git-moss/Push4Bitwig#220]
Hi, today I tried to fire up my push 1 with Bitwig 2.0 for the first time, but the controller is not working at all. It will not be auto-detected inside bitwig and if I try to add it manually it still only shows the "start live..." text in its display. It is not reacting at all. I'm on ubuntu 16.04 but until the update to 2.0 everthing worked as expected? Any ideas how to change this? Thanks indeed.

**Comment by git-moss** [comment]
Have you installed the latest v2 compatible script? Does it work when you add it manually?

**Comment by JohannesTress** [comment]
Yes, I downloaded the latest script from here... I can select the push controller via manual assignment in the dashboard, but the controller does not respont at all. I tried it with alsa/jack and pulsaaudio driver and all sorts of midi bridges I know - but without luck...

**Comment by git-moss** [comment]
Have you selected the SECOND Midi in-/output of the Push?
Do you see any errors in the Script console?

**Comment by JohannesTress** [comment]
ok. I assigned both midi in/outs and the push shows up as expected - but I can't controll anything. If I open up a project the controller shows the related tracks and data, but neither the pads nor the encoders work when touched. There's even no visual feedback at all. When I press play (via mouse) the controller shows the played notes in a marked clip.

**Comment by git-moss** [comment]
If I understand correctly, the display and buttons show their states correctly but you pressing buttons and turning knobs has no effect?! Maybe try, the other input port.
Have you checked the Script console output for errors?

**Comment by JohannesTress** [comment]
Yes that's correct. I have no errors in the script console and tried every possible combination of ins and outs... I'll try to reinstall the software, maybe this will solve the issue...

Am 5. Mai 2017 12:17:06 MESZ schrieb git-moss <notifications@github.com>:

**Comment by JohannesTress** [comment]
Ok. Reinstalled Bitwig and no it is working - but only if I switch Input 1&2 in the Controller Settings. The Second Push port is assigned to Midi in and the first on to midi out in the dashboard!

**Comment by git-moss** [comment]
That's great! Can you please send me the names of all Push ports appearing and which ones are the working ones?

**Comment by JohannesTress** [comment]
I see only 1 and 2. Those are reversed.
